plugins/modules/proxmox_tasks_info.py

```diff
@@ -0,0 +1 @@
+plugins/modules/cloud/misc/proxmox_tasks_info.py
```

Comment:
This doesn't look like a correct relative symlink from plugins/modules to plugins/modules/cloud/misc/proxmox_tasks_info.py.